### PR TITLE
GH-4: add new issue templates

### PR DESCRIPTION
**Pull request by rain1024**

### CODE REVIEW

**Comment by rain1024** [comment]
@quang-ph Please review this PR

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    attributes:
+      label: Who can help?
+      description: |
+        Your issue will be replied to more quickly if you can figure out the right person to tag with @
+        If you know how to use git blame, that is the easiest way, otherwise, here is a rough guide of **who to tag**.
+        Please tag fewer than 3 people.
+        
+        Models:
+
+          - ALBERT, BERT, XLM, DeBERTa, DeBERTa-v2, ELECTRA, MobileBert, SqueezeBert: `@LysandreJik`
```
Comment (quang-ph) [inline .github/ISSUE_TEMPLATE/bug-report.yml:24]: Here is the list of people to be tagged in an issue. Should be changed to correct contributors

Comment (rain1024) [reply .github/ISSUE_TEMPLATE/bug-report.yml:24]: @quang-ph I just remove some not neccessary fields. Please review it

**Review by quang-ph — APPROVED** [review]
Looks good to me

**Pull request merged**